app/models/series.rb

```diff
@@ -18,6 +18,7 @@ class Series < BaseModel
   has_many :stories, -> { order('episode_number DESC, position DESC, published_at DESC') }
   has_many :schedules
   has_many :audio_version_templates
+  has_many :distributions, as: :distributable
```

Comment:
I'm not renaming it, I'm using the polymorphic relationship to the distribution.
I am perhaps over engineering, but I expect some forms of distribution may be at an account level instead of the series, so not only series can be `distributable`, i.e. own distributions.